### PR TITLE
Sanitazers

### PR DESCRIPTION
**Pull request by DenisZackharov**
Change edit user atributes feature with sanitazers, and connect user decorator

### CODE REVIEW

File: app/controllers/concerns/authentication.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        super.decorate if user_signed_in?
+      end
+    end
+  end
+
+  private
+
+  def devise_parameter_sanitizer
+    if resource_class == User
+      ::UserParameterSanitazer.new(User, :user, params)
```
Comment (sergeyantonov1) [inline app/controllers/concerns/authentication.rb]:
```suggestion
      ::UserParameterSanitizer.new(User, :user, params)
```

File: app/controllers/concerns/authorization.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+  included do
+    include Pundit
+    rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
+  end
+
+  private
+
+  def user_not_authorized
+    flash[:alert] = "You are not authorized to perform this action."
```
Comment (sergeyantonov1) [inline app/controllers/concerns/authorization.rb]: We can use locales here, see documentation here https://guides.rubyonrails.org/i18n.html

File: app/decorators/user_decorator.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+class UserDecorator < ApplicationDecorator
+  delegate :id, :firstname, :lastname, :email
+
+  def firstname_lastname_with_email
```
Comment (DenisZackharov) [inline app/decorators/user_decorator.rb]: Fullname_with_email

File: config/database.yml [hunk 4]
```diff
@@ -3,6 +3,7 @@ default: &default
   adapter: postgresql
   encoding: unicode
   username: <%= ENV.fetch("DB_USERNAME", "postgres") %>
+  password: <%= ENV.fetch("DB_PASSWORD", "100100") %>
```
Comment (DenisZackharov) [inline config/database.yml]: In my home computer without this not working

Comment (sergeyantonov1) [reply config/database.yml]: @DenisZackharov it looks like not securable. You store your secret keys in github. You can set `DB_PASSWORD` in `.env` file
```suggestion
  password: <%= ENV.fetch("DB_PASSWORD") %>
```

```
# .env
DB_PASSWORD=100100
```

**Review by sergeyantonov1 — APPROVED** [review]

**Pull request merged**